src/app/pages/services/services.component.ts

```diff
@@ -177,13 +140,17 @@ export class ServicesComponent implements EntityTableConfig, OnInit {
 
   updateService(rpc: 'service.start' | 'service.stop', service: ServiceRow): void {
     let waiting = true;
+    this.cdr.markForCheck();
     // Delay spinner for fast API responses
     setTimeout(() => {
       if (waiting) service.onChanging = true;
+      this.cdr.markForCheck();
```

Comment:
I think in the future we may want to consider not mutating / extending original service, but instead have a separate `Map` of service name that is being loaded.